docs/Coiote_IoT_DM/firmware_update.md

```diff
@@ -0,0 +1,188 @@
+    
+    `state 2` **Downloaded** - The firmware download is completed
+    
+    `state 3` **Updating** - The Client starts updating its firmware, after which it changes its state back to *Idle*
+
+=== "**Update Results**"
+
+    `update result 1` **Firmware updated successfully**
+
+    `update result 2` **Insufficient flash memory for the new firmware package**
+
+    `update result 3` **Out of RAM during downloading proces**
+
+    `update result 4` **Connection lost during downloading process**
+
+    `update result 5` **Integrity check failure for new downloaded package**
+
+    `update result 6` **Unsupported package type**
+
+    `update result 7` **Invalid URI**
+
+    `update result 8` **Firmware update failed**
+
+    `update result 9` **Unsupported protocol**
+
+    `update result 10` **Firmware update cancelled**
+
+    `update result 11` **Firmware update deferred**
+
+
+!!! info
+    Learn more about the Firmware Update Object in the <a href="https://devtoolkit.openmobilealliance.org/OEditor/LWMOView?url=https%3A%2F%2Fraw.githubusercontent.com%2FOpenMobileAlliance%2Flwm2m-registry%2Fprod%2F5.xml" target="_blank">**OMA LwM2M Object and Resource Registry**</a>.
+
+
+## Prepare the Firmware Update
+
+1. Select the device you want to update from the [**Coiote Device Inventory**](https://eu.iot.avsystem.cloud/ui/device/inventory).
+    
+1. Select **Data model** from the top menu and validate if the Firmware Update object `/5` is visible, indicating that the Object is supported by the LwM2M Client.
+
+    ![Firmware update object](images/State 0 - fota object.png)
+
+1. Select **Firmware update** from the top menu and click **Update Firmware** from the top-right corner.
+
+    ![Update Firmware Tab](images/Device Inventory2.png)
+
+1. Select **Basic Firmware Update**
+
+    ![Basic Firmware Update](images/Basic Firmware Update.png)
+
+1. Choose between **Pull** and **Push**
+
+    **Pull method** (recommended): The LwM2M Client receives the URI of the file that is to be downloaded and pulls the file from it. 
+    
+    **Push method**: The LwM2M Server pushes the firmware file to the device.
+
+
+    !!! Info
+        **Pull** supports any of the following **transport types**:
+
+        - `CoAP` or `CoAPs` over `UDP`
+        - `CoAP` or `CoAPs` over `TCP`
+        - `HTTP` or `HTTPs`
+
+        **Push** uses `CoAPs` over `UDP` by default.
+
+    !!! Note
+        **Which transport protocol to choose?**
+        
+        Downloads using `CoAP(s)` over `UDP` tend to be slow due to limitation of the maximum CoAP Block size of 1024 bytes and the required acknowledgements for each Block transfer.
+
+        Choosing `CoAP(s)` over `TCP` or `HTTP(s)` usually results in faster download speeds.
+
+
+    ![Basic Firmware Update](images/Pull.png)
+
+
+1. Click **Schedule Update** from the bottom-right corner to trigger the Firmware Update process.
+
+    ![Schedule Update](images/schedule.png)
```

Comment:
This image is not even remotely to scale compared to the `Pull.png` screenshot above, which IMHO looks a bit comical and may mislead readers as to how the button actually looks like. I'd consider resizing them to scale.

Tip: If you're taking screenshots on a Mac with a Retina display, the screenshots will be 200% size compared to their declared pixel dimensions (regardless of system display scaling settings and assuming that the browser is set to 100% zoom). Scaling them back to exactly 50% the taken size (or taking them with the browser set to 50% zoom in the first place) should produce images that will look "actual size" when pasted into the document.

Reply:
Good point, updated.